src/node_options.cc

```diff
@@ -613,6 +613,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "path to watch",
             &EnvironmentOptions::watch_mode_paths,
             kAllowedInEnvvar);
+  AddOption("--preserve-output",
```

Comment:
I think the name should be specific to watch mode (e.g., `--watch-preserve-output`). If we keep adding modes (normal, eval, watch, test, ...) to Node.js, it will eventually become difficult to make sense of flags that only affect a specific mode unless their name is related to the mode.

Reply:
Understood renaming the same

Reply:
have refactored @tniessen